### PR TITLE
Fusion Sep release: 1.0.5

### PR DESCRIPTION
**Pull request by pure-qwei**
This is for fusion September release with release branch: release-20220920.  Version number: 1.0.5

### CODE REVIEW

File: fusion/configuration.py [hunk 1]
```diff
@@ -249,5 +249,5 @@ def to_debug_report(self):
                "OS: {env}\n"\
                "Python Version: {pyversion}\n"\
                "Version of the API: 1.0\n"\
-               "SDK Package Version: 1.0.3".\
+               "SDK Package Version: 1.0.0".\
```
Comment (maoxianxian) [inline fusion/configuration.py:252]: 1.0.5?

**Comment by maoxianxian** [comment]
Could you update the `SDK Package Version` in `build/lib.linux-x86_64-2.7/fusion/configuration.py` as well?

**Comment by pure-cye** [comment]
Could you update the `SDK Package Version` in `build/lib.linux-x86_64-2.7/fusion/configuration.py` as well?

File: build/lib/fusion/configuration.py [hunk 2]
```diff
@@ -249,5 +249,5 @@ def to_debug_report(self):
                "OS: {env}\n"\
                "Python Version: {pyversion}\n"\
                "Version of the API: 1.0\n"\
-               "SDK Package Version: 1.0.4".\
+               "SDK Package Version: 1.0.0".\
```
Comment (pure-cye) [inline build/lib/fusion/configuration.py]: 1.0.5?

File: fusion/configuration.py (hunk 1 above)
Comment (pure-alou) [inline fusion/configuration.py]: 1.0.5

**Review by pure-alou — CHANGES_REQUESTED** [review]

**Comment by pure-cye** [comment]
Looks like the entire `lib.linux-x86_64-2.7` directory is nuked. Is there a reason for that?

**Comment by pure-qwei** [comment]
> Looks like the entire `lib.linux-x86_64-2.7` directory is nuked. Is there a reason for that?

@pure-cye  I just rerun python3 setup.py bdist_wheel,  and `lib.linux-x86_64-2.7` get deleted,  instead we have `lib.linux-x86_64`.  I think it is because we are using python3 instead of python2.7 ?

**Comment by pure-cye** [comment]
> > Looks like the entire `lib.linux-x86_64-2.7` directory is nuked. Is there a reason for that?
> 
> @pure-cye I just rerun python3 setup.py bdist_wheel, and `lib.linux-x86_64-2.7` get deleted, instead we have `lib.linux-x86_64`. I think it is because we are using python3 instead of python2.7 ?

I see. Make sense

**Comment by pure-cye** [comment]
LGTM. I don't have permission to approve so I'll leave it to others

**Review by pure-alou — APPROVED** [review]

**Pull request merged**